combat-trainer.lic

```diff
@@ -715,6 +715,8 @@ class SpellProcess
       end
     end
 
+    waitrt? while XMLData.prepared_spell == 'Vivisection' && game_state.use_stealth_attack? && ! hide?
```

Comment:
The `hide?` helper also takes case of waitrt, so I don't think you need to do that here.